scenarios/json.benchmarks.yml

```
@@ -33,6 +34,18 @@ jobs:
     readyStateText: Application started.
     arguments: "--urls {{serverScheme}}://{{serverAddress}}:{{serverPort}}"
 
+  jsonbenchmarks:
+    source:
+      repository: https://github.com/aspnet/benchmarks.git
+      branchOrCommit: main
+      project: src/BenchmarksApps/Json/Json.csproj
+    readyStateText: Application started.
+    variables:
+      protocol: http
+      server: Kestrel
+      scenario: json
+    arguments: "--nonInteractive true --scenarios {{scenario}} --urls {{serverScheme}}://{{serverAddress}}:{{serverPort}} --server {{server}} --protocol {{protocol}}"
```

```suggestion
    variables:
    arguments: "--urls {{serverScheme}}://{{serverAddress}}:{{serverPort}}"
```

I don't think these are used, probably a copy from the main Benchmarks project.